package Results;


message Consumption {
	required int32 nActiveCores = 1;
	required double meanPower = 2;
	required double alpha = 3;
};


message Execution {
	required int32 TSA = 1;					//
	required int32 CSA = 2;					//
	required bool feasible = 3;				//
	repeated Consumption sfa = 4;
	required Consumption no_sfa = 5;
	required string partitioningFile = 6;	//
	required int32 n = 7;
	required double u = 8;
};


message Result {
	repeated Execution iterations = 1;
	required string hwFile = 2;
};

